packages/analytics/dashboard-renderer/src/types/dashboard-renderer-types.ts

```diff
@@ -55,6 +55,14 @@ const allowCsvExport = {
   type: 'boolean',
 } as const
 
+const goToExplore = {
```

Comment:
I don't think either of these properties should be exposed in the schema.  The things in the schema are eventually going to be part of the public API for dashboards; there's no case where an end user should be able to directly configure these props.

I'd recommend:

- `editable` is a new property exposed via `context`.  I think it's OK (especially for now) to say that either all tiles are editable or none are.
- `goToExplore` is probably a base URL that's configured via the analytics bridge, and the query data provider calculates the final URL to hand to `AnalyticsChart`.

Reply:
How does the dashboard know if the tiles should have go to explore or not? 🤔